stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/ListenerContainer.kt

```diff
@@ -0,0 +1,21 @@
+package io.getstream.chat.android.ui.messages.adapter
+
+import io.getstream.chat.android.ui.messages.view.MessageListView.AttachmentClickListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.GiphySendListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.MessageClickListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.MessageLongClickListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.MessageRetryListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.ReactionViewClickListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.ReadStateClickListener
+import io.getstream.chat.android.ui.messages.view.MessageListView.UserClickListener
+
+public interface ListenerContainer {
```

Comment:
Created separate `ListenerContainer`, `MessageListItemDiffCallback` and `MessageListItemPayloadDiff` classes for the new module, so that they can be independent of the old UI's implementations of these